__tests__/classes/CanvasRenderingContext2D.__getClippingPath.js

```diff
@@ -51,4 +51,26 @@ describe('__getClippingRegion', () => {
     ctx.restore();
     expect(region).toStrictEqual(ctx.__getClippingRegion());
   });
+
```

Comment:
These tests are good, but the test file is setup so that the "snapshots" automatically test the context state.

Perhaps we could consolidate this to a single test?

```ts
  it('should restore the clipping region correctly when restored', () => {
    const region = ctx.__getClippingRegion();
    ctx.arc(1, 2, 3, 4, 5);
    ctx.save();
    ctx.rect(1, 2, 3, 4);
    ctx.arc(1, 2, 3, 4, 5);
    ctx.clip();
    ctx.restore();
  });
```

We could simply expect the clipping region to be a single item (which accurately tests the methods.)